app/controllers/api/v2/courses_controller.rb

```diff
@@ -21,6 +21,34 @@ def new
         render jsonapi: @course, include: params[:include]
       end
 
+      # For creating and validating a course model without persisting it to the database.
+      # Used by the new course wizard in frontend.
```

Comment:
I've gone off this comment, code seems pretty clear in the end

Reply:
Delete it!